features/admin.users.v1/components/wizard/steps/add-user-basic.tsx

```diff
@@ -367,6 +582,227 @@ export const AddUserUpdated: React.FunctionComponent<AddUserProps> = (
+                                        schema.name,
+                                        multiValuedAttributeValues[schema.name],
+                                        multiValuedInputFieldValue[schema.name]
+                                    )
+                                };
+                            } else {
+                                // Handle emails and phoneNumbers and their sub attributes.
+                                // Extract the sub attributes from the form values.
+                                for (const value of values.keys()) {
+                                    const subAttribute: string[] = value.split(".");
+
+                                    if (subAttribute[0] === schemaNames[0]) {
+                                        attValues.set(value, values.get(value));
+                                    }
+                                }
+
+                                for (const [ key, value ] of attValues) {
+                                    const attribute: string[] = key.split(".");
+
+                                    if (value && value !== "") {
+                                        if (attribute.length === 1) {
+                                            attributeValues.push(value);
+                                        } else {
+                                            attributeValues.push({
+                                                type: attribute[1],
+                                                value: value
+                                            });
+                                        }
+                                    }
+                                }
+                                opValue = {
+                                    [schemaNames[0]]: attributeValues
+                                };
+                            }
+                        }
+                    } else {
+                        if (schemaNames.length === 1) {
+                            if (schema.extended) {
+                                const schemaId: string = schema?.schemaId
+                                    ? schema.schemaId
+                                    : userConfig.userProfileSchema;
+
+                                if (schema.name === "externalId") {
+                                    opValue = {
+                                        [schemaNames[0]]: values.get(schemaNames[0])
+                                    };
+                                } else {
+                                    opValue = {
+                                        [schemaId]: {
+                                            [schemaNames[0]]: schema.type.toUpperCase() === "BOOLEAN" ?
+                                                !!values.get(schema.name)?.includes(schema.name) :
+                                                values.get(schemaNames[0])
```

Comment:
Formatting suggestion
```suggestion
                                            [schemaNames[0]]: schema.type.toUpperCase() === "BOOLEAN"
                                                ? !!values.get(schema.name)?.includes(schema.name)
                                                : values.get(schemaNames[0])
```

Reply:
Fixed with 5cbff8089174971c91a1c1aabd4c1a11270bdb83